### DISCUSSION TITLE
Configuration Sources Filters

### DISCUSSION THREAD

**Discussion by qcxiang** [DNSCrypt/dnscrypt-proxy#2397] [Ideas]
希望能支持这样的功能：
支持过滤掉延迟超过100ms的server
或者只取延迟最低的5个server.

Hope to support something like this:
Support filtering out servers with a delay exceeding 100ms
Or just take the 5 servers with the lowest delay.

**Comment by lifenjoiner** [comment]
Learn this first: https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Load-Balancing-Options#load-balancing-options
